myosuite/physics/mj_sim_scene.py

```diff
@@ -16,6 +17,7 @@
 import_utils.dm_control_isavailable()
 import_utils.mujoco_isavailable()
 import dm_control.mujoco as dm_mujoco
+from dm_control.mujoco.wrapper import MjModel as dm_MjModel
```

Comment:
We should get rid of dm_control dependency and directly with with mujoco

Reply:
I agree, but at the moment we depend on it, and on mujoco_py too. The base env classes expect the dm_control classes and break with the proper mujoco ones. I think the switch is absolutely necessary, but should be handled in a separate PR